<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSimpleMarker</class>
 <widget class="QWidget" name="WidgetSimpleMarker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>28</number>
     </property>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Offset X,Y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="spinAngle">
       <property name="suffix">
        <string> Â°</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="spinOffsetX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spinOffsetY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mOffsetUnitComboBox">
         <item>
          <property name="text">
           <string>Millimeter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Map unit</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="mDataDefinedPropertiesButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Data defined properties...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mOutlineWidthLabel">
       <property name="text">
        <string>Outline width</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDoubleSpinBox" name="mOutlineWidthSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mOutlineWidthUnitComboBox">
         <item>
          <property name="text">
           <string>Millimeter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Map unit</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDoubleSpinBox" name="spinSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mSizeUnitComboBox">
         <item>
          <property name="text">
           <string>Millimeter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Map unit</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fill color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsColorButton" name="btnChangeColorFill">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Border color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsColorButton" name="btnChangeColorBorder">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="lstNames">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="gridSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QPushButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lstNames</tabstop>
  <tabstop>spinOffsetX</tabstop>
  <tabstop>spinOffsetY</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
